src/Action/SetObjectFieldValueAction.php

```diff
@@ -158,6 +158,7 @@ public function __invoke(Request $request): JsonResponse
 
         // render the widget
         // todo : fix this, the twig environment variable is not set inside the extension ...
+        /** @var SonataAdminExtension $extension */
```

Comment:
I've added phpstan-symfony plugin, It doesn't help. Added assert

Reply:
I'm removing it, since it requires container.xml

`You have to provide a path to srcDevDebugProjectContainer.xml` https://github.com/phpstan/phpstan-symfony#configuration

and also breaks CI build